server/game/cards/08.1-TAK/DaarioNaharis.js

```diff
@@ -7,7 +7,7 @@ class DaarioNaharis extends DrawCard {
                 afterChallenge: event => event.challenge.winner === this.controller && this.isAttacking()
             },
             target: {
-                cardCondition: card => card.location === 'play area' && card !== this && (card.kneeled || card.controller !== this.controller) &&
+                cardCondition: card => card.location === 'play area' && card !== this &&
```

Comment:
Slight additional change that needs to be made - in the handler function, the take control effect is wrapped in an if checking that the controllers aren't equal. That if should be removed as well since the take control effect should always be applied.